.changeset/updated-dependency-version.md

```
@@ -0,0 +1,61 @@
+---
+"eridu-tech": patch
+---
+
+Updated dependency versions.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)